odoo/http.py

```diff
@@ -1316,22 +1315,15 @@ def load_addons(self):
         controllers and configure them.  """
         # TODO should we move this to ir.http so that only configured modules are served ?
         statics = {}
-        manifests = addons_manifest
         for addons_path in odoo.addons.__path__:
-            for module in sorted(os.listdir(str(addons_path))):
-                if module not in manifests:
-                    # Deal with the manifest first
-                    mod_path = opj(addons_path, module)
-                    manifest = read_manifest(addons_path, module)
-                    if not manifest or (not manifest.get('installable', True) and 'assets' not in manifest):
-                        continue
-                    manifest['addons_path'] = addons_path
-                    manifests[module] = manifest
-                    # Then deal with the statics
-                    path_static = opj(addons_path, module, 'static')
-                    if os.path.isdir(path_static):
-                        _logger.debug("Loading %s", module)
-                        statics['/%s/static' % module] = path_static
+            for module in os.listdir(addons_path):
+                manifest = load_manifest(module)
+                static_path = opj(addons_path, module, 'static')
```

Comment:
opj, I personnaly hate this name

opj ? what is that? Some kind of black magic? Ho, os.path.join. Nevermind.

Reply:
opj is used everywhere in the source code